nest-cli/src/commands/pull.rs

```diff
@@ -0,0 +1,50 @@
+use failure::{format_err, Error, ResultExt};
+use indicatif::{ProgressBar, ProgressStyle};
+use libnest::config::Config;
+use libnest::transaction::{PullTransaction, Transaction};
+
+use super::download::Download;
+use super::{ask_confirmation, print_transactions};
+
+pub fn pull(config: &Config) -> Result<(), Error> {
+    let transactions: Vec<_> = config
+        .repositories()
+        .into_iter()
+        .map(|repository| Transaction::Pull(PullTransaction::from(repository)))
+        .collect();
+
+    print_transactions(&transactions);
+
+    if !ask_confirmation("Would you like to apply these transactions ?", true)? {
```

Comment:
What if there is only one transaction?
You handled proper-English so far, so I feel like I should warn you when you forget :p 

(Same below)